index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio</title>
+</head>
+<body>
+    <div class="interface">
+        <header class="header">
+            <img class="logo" src="FireBorne Entertainment.png" alt="My Logo">
+            <img class="hamburger" src="./Union.png" alt="Hamburger">
+        </header>
```

Comment:
Good job by using the header to wrap the upper part of your page , and I would suggest that you wrap the humberger icon in a nav tag as it is meant to be a navigation menu "this functionality will be added later on" using nav is good for semantic reasons